### ISSUE TITLE
Running only as a Dat server

### ISSUE THREAD

**Issue by AlbertoElias** [beakerbrowser/dathttpd#33]
Is it possible to use `dathttpd` only as a Dat server, that would make it work for my custom domain under the Dat protocol, but leaves handling http/https connections to a custom server? Thank you!

**Comment by pfrazee** [comment]
Hey @AlbertoElias 

You can disable the https hosting with the [`datOnly`](https://github.com/beakerbrowser/dathttpd#siteshostnamedatonly) config. It's not currently possible to stop the http from starting entirely (PRs accepted) but if you also set [`letsencrypt`](https://github.com/beakerbrowser/dathttpd#letsencrypt) to false, then you can just set the [`ports.http`](https://github.com/beakerbrowser/dathttpd#portshttp) to something random and the server would never be hit.

The `dathttpd` hosting on https normally handles DNS shortnames for you. If you disable the https hosting but still want shortnames, you'll need to manually setup the .well-known file. See https://github.com/beakerbrowser/beaker/wiki/Authenticated-Dat-URLs-and-HTTPS-to-Dat-Discovery#proposal.

**Comment by AlbertoElias** [comment]
Thanks a lot for the detailed explanation.

For that use case, would dathttpd be worth it? Or just running a service that runs Dat and using a .well-known file be a better option?

**Comment by pfrazee** [comment]
Yeah it's probably still a good choice but it's up to you. It'll work as well as any alternative.
